panel/src/components/Forms/Field/LinkField.vue

```diff
@@ -345,6 +349,24 @@ export default {
 				return null;
 			}
 		},
+		selectPage(page) {
+			if (page.uuid) {
+				this.onInput(page.uuid);
+				return;
+			}
+
+			this.switchType("url");
+			this.onInput(page.url);
+		},
+		selectFile(file) {
+			if (file.uuid) {
+				this.onInput(file.uuid);
+				return;
+			}
+
+			this.switchType("url");
+			this.onInput(file.url);
+		},
```

Comment:
This code block can be simplify like that

```suggestion
		selectModel(model) {
			if (model.uuid) {
				this.onInput(model.uuid);
				return;
			}

			this.switchType("url");
			this.onInput(model.url);
		},
```